### PR TITLE
add readme doc

### PR DESCRIPTION
**Pull request by Markuson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# HELLO WORLD
+----
+
+## Intro
```
Comment (manuelbarzi) [inline README.md]: @Markuson "Intro no", but "Introduction" yes

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+TODO explain project
```
Comment (manuelbarzi) [inline README.md]: @Markuson hey, you should finish this TODO

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+## Functional description
+
+TODO explain functional description
```
Comment (manuelbarzi) [inline README.md]: @Markuson hey, you should finish this TODO

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+## Technical description
+
+TODO explain technical descnription
```
Comment (manuelbarzi) [inline README.md]: @Markuson hey, you should finish this TODO

File: README.md (hunk 1 above)
Comment (Markuson) [reply README.md]: @manuelbarzi  sorry but "intro" it's ok for me.

**Pull request merged**